ndk/src/font.rs

```diff
@@ -0,0 +1,401 @@
+//! Bindings for [`AFont`], [`AFontMatcher`], and [`ASystemFontIterator`]
+//!
+//! [`AFont`]: https://developer.android.com/ndk/reference/group/font#afont_close
+//! [`AFontMatcher`]: https://developer.android.com/ndk/reference/group/font#afontmatcher_create
+//! [`ASystemFontIterator`]: https://developer.android.com/ndk/reference/group/font#asystemfontiterator_open
+
+use std::convert::TryFrom;
+use std::ffi::{CStr, OsStr};
+use std::fmt;
+use std::os::unix::prelude::OsStrExt;
+use std::path::Path;
+use std::ptr::NonNull;
+
+/// An integer holding a valid font weight value between 1 and 1000.
+///
+/// See the followings for more details:
+/// * [`AFONT_WEIGHT_*`]
+/// * [`Font::weight`]
+///
+/// [`AFONT_WEIGHT_*`]: https://developer.android.com/ndk/reference/group/font#anonymous-enum-33
+#[derive(Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Debug)]
+pub struct FontWeight(u16);
+
+impl FontWeight {
+    pub const fn new(value: u16) -> std::result::Result<Self, TryFromU16Error> {
+        if 0 < value && value <= 1000 {
```

Comment:
Use existing `MIN` and `MAX` consts instead of open-coding this.

Reply:
Fixed! Please see below :)